workflows/mag.nf

```diff
@@ -474,6 +475,14 @@ workflow MAG {
             )
             ch_versions = ch_versions.mix(BIN_SUMMARY.out.versions)
         }
+        if (params.generate_bigmag_file && !params.skip_binqc && params.run_gunc && params.run_checkm2 && params.run_busco && !params.skip_gtdbtk && !params.skip_quast) {
```

Comment:
I think we can simplify this to 


```suggestion
        if (params.generate_bigmag_file ) {
```

And have an upstream validation check on pipeline initalisation to error if all the binners are not executed correctly

You can add this check here: https://github.com/jeffe107/mag/blob/a670f5e77c1bebc47f30730fa3a2d46cd60fe8d5/subworkflows/local/utils_nfcore_mag_pipeline/main.nf#L278-L399